hasura/metadata/databases/default/tables/public_memberships.yaml

```diff
@@ -0,0 +1,58 @@
+table:
+  name: memberships
+  schema: public
+object_relationships:
+  - name: team
+    using:
+      foreign_key_constraint_on: team_id
+  - name: user
+    using:
+      foreign_key_constraint_on: user_id
+insert_permissions:
+  - role: service
```

Comment:
why does service need insert permissions?

Reply:
for signup endpoint